elements/rh-table/rh-sort-button.ts

```diff
@@ -0,0 +1,70 @@
+import { LitElement, html, svg } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { property } from 'lit/decorators/property.js';
+import styles from './rh-sort-button.css';
+
+const DIRECTIONS = { asc: 'desc', desc: 'asc' } as const;
+
+export class RequestSortEvent extends Event {
+  constructor(
+      public direction: 'asc' | 'desc',
+  ) {
+    super('request-sort', {
+      bubbles: true,
+      cancelable: true,
+    });
+  }
+}
```

Comment:
I think you could use `ComposedEvent` here

`import { ComposedEvent } from '@patternfly/pfe-core';`

```Suggestion
export class RequestSortEvent extends ComposedEvent {
  constructor(
      public direction: 'asc' | 'desc',
  ) {
    super('request-sort', { cancelable: true });
  }
}
```